boucle.py

```diff
@@ -4,18 +4,19 @@
 
 import jack
 import sys
-from clip import Clip, Song
+from clip import Clip, Song, getDefaultOutputNames
 from gui import Gui
 from PyQt5.QtWidgets import QApplication
 from queue import Empty
 
-song = Song(8, 8)
+song = Song(8, 8, getDefaultOutputNames(8))
 
 client = jack.Client("Super Boucle")
 midi_in = client.midi_inports.register("input")
 midi_out = client.midi_outports.register("output")
-outL = client.outports.register("output_L")
-outR = client.outports.register("output_R")
+outL = client.outports.register("MAIN_L")
+outR = client.outports.register("MAIN_R")
+
 inL = client.inports.register("input_L")
 inR = client.inports.register("input_R")
 
```

Comment:
Also if we stick to main being the only port whose routing is managed by superboucle:
I suppose autoconnect must happen on song-load, because when there are songs that do not have the main port, it will be removed.